index.html

```diff
@@ -5,38 +5,183 @@
+          <li>
+            <img src="./icons/ceremonia-de-apertura.png" alt="">
+            <span>Main Event</span>
+            <p>You can listen to all the bands Friday and Saturday from 4 pm and we will finish at 11 pm.</p>
+          </li>
+          <li>
+            <img src="./icons/ceremonia-de-apertura.png" alt="">
+            <span>Closure</span>
+            <p>Each of the bands in a special place will be autographing your favorite memories.</p>
+          </li>
+        </ul>
+        <a class="btn" href="#">Join to Concert of the Century</a>
+        <a class="btn-program" href="#main-page">SEE THE WHOLE PROGRAM</a>
+      </section>
+      <div id="featured">Featured Bands</div>
+      <section class="bands" id="bands" >
+        
+      </section>
+      <footer>
+        <h3>Partner</h3>
+        <ul class="sponsors">
+          <li><a href="#"><img src="./images/mozilla-logo.png" alt=""></a></li>
+          <li><a href="#"><img src="./images/google-logo.svg" alt=""></a></li>
+          <li><a href="#"><img src="./images/naver-logo.png" alt=""></a></li>
+          <li><a href="#"><img src="./images/daumkakao-logo.png" alt=""></a></li>
+          <li><img src="./images/airbnb-logo.svg" alt=""></li>
+        </ul>
+      </footer>
+      <ul class="last-section">
+        <li>s
+          <img src="./images/logo-rock2.jpg" alt="">
+        </li>
+        <li>
+          <p>All content on this website, unless otherwise specified, is available under a Creative Commons Atrribution<br>Copyright © 2022. <br>All rights reserved.</p>
+        </li>
+      </ul>
+  </section>
+  <section class="about" id="about">
+    <header class="header__about">
+      <span class="upper-menu">
+        <span class="upper-menu__icons">
+          <a href="#" class="bi bi-facebook"></a>
+          <a href="#" class="bi bi-twitter"></a>
+        </span>
+        <span class="upper-menu__links">
+          <a href="#">English</a>
+          <a href="#">My Page</a>
+          <a href="#">Logout</a>
+        </span>
+      </span>
+      <div class="links">
+        <a class="logo" href="#"><img src="./images/logo-rock2.jpg" alt="header icon"></a>
+        <nav>
+          <ul class="menu__list">
+            <li class="menu__item menu__about"><a href="#">About</a></li>
+            <li class="menu__item"><a href="#">Program</a></li>
+            <li class="menu__item"><a href="#">Join</a></li>
+            <li class="menu__item"><a href="#">Sponsor</a></li>
+            <li class="menu__item"><a href="#">News</a></li>
+            <li class="menu__item mark"><a href="#">Century Concert</a></li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <a class="hamburger-menu__about" href="#"><img src="./icons/menu.png" alt="hamburger menu"></a>
+    <div class="about-general">
+      <h3>"Welcome music lovers"</h3>
+      <h1 class="about__title">THE CONCERT OF THE CENTURY <br>THE MOST SPECTACULAR MEETING OF POP AND ROCK IN 2022</h1>
```

Comment:
Hello Hammas. Thank you for your suggestions because I know you as a code reviewer wants we become the best developer. However, I was looking for the requeriment of the Capstone agree with your solitude and I did not find anything. My project has all the functionality agree with the Microverse requierements. Please check it.